cmd/testworkflow-init/main.go

```diff
@@ -369,7 +376,18 @@ func main() {
 				stopTimeoutWatcher := orchestration.WatchTimeout(finalizeTimeout, leaf...)
 
 				// Run the command
-				commands.Run(*action.Execute, currentContainer)
+				d := data.GetState()
+				recorderConfig := utilization.Config{
+					Dir:       "./.tk/metrics",
```

Comment:
It rather should be stored in `/.tktw` directory, along with other internal files (for sure not CWD)

Reply:
updated